Sources/SwiftDriver/Dependency Scanning/ModuleDependencyBuildGeneration.swift

```diff
@@ -16,8 +16,8 @@ import Foundation
 extension Driver {
   /// For the current moduleDependencyGraph, plan the order and generate jobs
   /// for explicitly building all dependency modules.
-  mutating func planExplicitModuleDependenciesCompile(dependencyGraph: InterModuleDependencyGraph)
-  throws -> [Job] {
+  mutating func planExplicitModuleDependenciesCompile(dependencyGraph: InterModuleDependencyGraph
```

Comment:
Nitpick: can you bring the argument down onto its own line?

```suggestion
  mutating func planExplicitModuleDependenciesCompile(
    dependencyGraph: InterModuleDependencyGraph
  ) throws -> [Job] {
```